<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DIO_Configurator</string>
  </property>
  <widget class="QGroupBox" name="configuration_group">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>631</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Configuration</string>
   </property>
   <widget class="QGroupBox" name="Mode_group">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>221</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Mode</string>
    </property>
    <widget class="QRadioButton" name="Output_button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Input_button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Output_config_group">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Config</string>
    </property>
    <widget class="QRadioButton" name="High_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>High</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Low_button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Low</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Input_config_group">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Input Config</string>
    </property>
    <widget class="QRadioButton" name="Pull_Up_button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pull Imp</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Pull_Imp_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pull Up</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="PinName_LineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter the pin name</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Default_Name_CheckBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Default Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="Output_Folder_Path_LineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>440</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Browse">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>440</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>440</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Output Folder Path:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SaveCurrent">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Current</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>501</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>279</width>
      <height>1256</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_Pins">
       <property name="title">
        <string>Pins</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Pin_0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Pin_1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Pin_2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Pin_3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_8">
          <property name="text">
           <string>Pin_4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_6">
          <property name="text">
           <string>Pin_5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_7">
          <property name="text">
           <string>Pin_6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>Pin_7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_12">
          <property name="text">
           <string>Pin_8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_10">
          <property name="text">
           <string>Pin_9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_11">
          <property name="text">
           <string>Pin_10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_9">
          <property name="text">
           <string>Pin_11</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_16">
          <property name="text">
           <string>Pin_12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_14">
          <property name="text">
           <string>Pin_13</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_15">
          <property name="text">
           <string>Pin_14</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_13">
          <property name="text">
           <string>Pin_15</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_20">
          <property name="text">
           <string>Pin_16</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_18">
          <property name="text">
           <string>Pin_17</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_19">
          <property name="text">
           <string>Pin_18</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_23">
          <property name="text">
           <string>Pin_19</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_21">
          <property name="text">
           <string>Pin_20</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_22">
          <property name="text">
           <string>Pin_21</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_17">
          <property name="text">
           <string>Pin_22</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_27">
          <property name="text">
           <string>Pin_23</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_25">
          <property name="text">
           <string>Pin_24</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_26">
          <property name="text">
           <string>Pin_25</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_24">
          <property name="text">
           <string>Pin_26</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_31">
          <property name="text">
           <string>Pin_27</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_29">
          <property name="text">
           <string>Pin_28</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_30">
          <property name="text">
           <string>Pin_27</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_28">
          <property name="text">
           <string>Pin_27</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_35">
          <property name="text">
           <string>Pin_28</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_33">
          <property name="text">
           <string>Pin_29</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_34">
          <property name="text">
           <string>Pin_30</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_32">
          <property name="text">
           <string>Pin_31</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>status</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_43">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_56">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_63">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_53">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_42">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_52">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_39">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_61">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_46">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_48">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_40">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_38">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_57">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_50">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_47">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_54">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_60">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_59">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_45">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_51">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_62">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_49">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_55">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_41">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_58">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_44">
          <property name="text">
           <string>not configured</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>91</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_FinalSave">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>510</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Configuration</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Output_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>Output_config_group</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Output_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>Input_config_group</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Input_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>Input_config_group</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Input_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>Output_config_group</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Default_Name_CheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>PinName_LineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Default_Name_CheckBox</sender>
   <signal>clicked()</signal>
   <receiver>PinName_LineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
